src/test/testRegister.js

```diff
@@ -0,0 +1,179 @@
+          .set("Authorization", "Bearer " + userMock.token.admin)
+          .send(registerMock.emailEmpty)
+          .end((err,res)=>{
+            expect(res).to.have.status(400);
+            done();
+          })
+      }); 
+
+      it("Should not create new user if gender is longer than eight chars ", done=>{
+        chai 
+          .request(app)
+          .post("/api/v1/auth/register")
+          .set("Authorization", "Bearer " + userMock.token.admin)
+          .send(registerMock.isBig)
+          .end((err,res)=>{
+            expect(res).to.have.status(400);
+            done();
+          })
+      }); 
+
+      it("Should not create new user if gender is not given ", done=>{
+        chai 
+          .request(app)
+          .post("/api/v1/auth/register")
+          .set("Authorization", "Bearer " + userMock.token.admin)
+          .send(registerMock.genderNotGiven)
+          .end((err,res)=>{
+            expect(res).to.have.status(400);
+            done();
+          })
+      }); 
+
+      it("Should not create new user if gender empty string ", done=>{
+        chai 
+          .request(app)
+          .post("/api/v1/auth/register")
+          .set("Authorization", "Bearer " + userMock.token.admin)
+          .send(registerMock.genderEmpty)
+          .end((err,res)=>{
+            expect(res).to.have.status(400);
+            done();
+          })
+      }); 
+
+      it("Should not create new user if no birth date given ", done=>{
+        chai 
+          .request(app)
+          .post("/api/v1/auth/register")
+          .set("Authorization", "Bearer " + userMock.token.admin)
+          .send(registerMock.birthDateNotGiven)
+          .end((err,res)=>{
+            expect(res).to.have.status(400);
+            done();
+          })
+      }); 
+
+      it("Should not create new user if no birthdate is shorter than six chars", done=>{
+        chai 
+          .request(app)
+          .post("/api/v1/auth/register")
+          .set("Authorization", "Bearer " + userMock.token.admin)
+          .send(registerMock.birthShort)
+          .end((err,res)=>{
+            expect(res).to.have.status(400);
+            done();
+          })
+      }); 
+
+      it("Should not create new user if no birthdate is invalid", done=>{
+        chai 
+          .request(app)
+          .post("/api/v1/auth/register")
+          .set("Authorization", "Bearer " + userMock.token.admin)
+          .send(registerMock.invalidDate)
+          .end((err,res)=>{
+            expect(res).to.have.status(400);
+            done();
+          })
+      }); 
+      it("Should not create new user if birthdate is empty", done=>{
+        chai 
+          .request(app)
+          .post("/api/v1/auth/register")
+          .set("Authorization", "Bearer " + userMock.token.admin)
+          .send(registerMock.emptyDate)
+          .end((err,res)=>{
+            expect(res).to.have.status(400);
+            done();
+          })
+      }); 
```

Comment:
You cannot have all these tests checking only for status code (400). Please add more checks, for example, the error messages in the response body.